command/ca/init.go

```diff
@@ -137,10 +162,32 @@ func initAction(ctx *cli.Context) (err error) {
 		return err
 	}
 
-	name, err := ui.Prompt("What would you like to name your new PKI? (e.g. Smallstep)",
-		ui.WithValidateNotEmpty(), ui.WithValue(ctx.String("name")))
-	if err != nil {
-		return err
+	var name string
+	if ra == "" {
+		name, err = ui.Prompt("What would you like to name your new PKI? (e.g. Smallstep)",
+			ui.WithValidateNotEmpty(), ui.WithValue(ctx.String("name")))
+		if err != nil {
+			return err
+		}
+	} else {
+		iss, err := ui.Prompt("What certificate authority would you like to use? (e.g. projects/ca/locations/us-west1/certificateAuthorities/intermediate-ca)",
+			ui.WithValidateNotEmpty(), ui.WithValue(ctx.String("issuer")))
+		if err != nil {
+			return err
+		}
+		file := ctx.String("credentials-file")
+		if !ctx.IsSet("credentials-file") {
```

Comment:
should we be forcing a credentials file entry here instead of just allowing it to be empty if you don't pass one in via flag?

Reply:
You don't need a credentials-file if you're using a service account. 

I want a user to be able to run this command without the prompt, if all parameters are passed, in this case `--credentials-file ""`

Reply:
Right, I don't understand why we need the prompt at all. If you don't pass `--credentials-file` then we just use the default one on the instance.

Reply:
I was imitating the rest of the parameters here. But yes, it can be confusing.